custom_components/plugwise/__init__.py

```diff
@@ -122,12 +122,8 @@ def async_migrate_entity_entry(entry: er.RegistryEntry) -> dict[str, Any] | None
         return {"new_unique_id": entry.unique_id.replace("-slave_boiler_state", "-secondary_boiler_state")}
     if entry.domain == Platform.SWITCH and entry.unique_id.endswith("-plug"):
         return {"new_unique_id": entry.unique_id.replace("-plug", "-relay")}
-    if entry.domain == Platform.SENSOR and entry.unique_id.endswith(
-        "-relative_humidity"
```

Comment:
expecting this to be formatting? or would we want to partial the strings in this section as well?

Reply:
This used to be the result of running Black. But the added line was longer and not formatted by Ruff, so I removed this formatting as a test. I didn't get reformatted by Ruff.